lk2nd/device/dts/msm8226/rules.mk

```diff
@@ -1,5 +1,5 @@
 # SPDX-License-Identifier: BSD-3-Clause
 LOCAL_DIR := $(GET_LOCAL_DIR)
 
-DTBS += \
```

Comment:
I guess removing this wasn't intended?

Reply:
I don't recall exactly why I removed it, but can I have both DTBS and this QCDTBS in the makefile?

Reply:
Yes.

Reply:
Appears to work, thanks! :)